README.md

```diff
@@ -7,7 +7,7 @@ Anserini
 Build using Maven:
 
 ```
-mvn clean package appassembler:assemble
+mvn clean package
 ```
 
 The `eval/` directory contains evaluation tools and scripts, including `trec_eval`. Before using `trec_eval`, unpack and compile it.
```

Comment:
Can we add more specific commands here?
`tar xvfz trec_eval.9.0.tar.gz && cd trec_eval.9.0 && make`